eidas-saml-parser/src/test/java/uk/gov/ida/notification/eidassaml/apprule/base/TestMetatronResource.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.ida.notification.eidassaml.apprule.base;
+
+
+import uk.gov.ida.notification.contracts.CountryMetadataResponse;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import java.net.URI;
+
+import static uk.gov.ida.saml.core.test.TestCertificateStrings.TEST_RP_PUBLIC_ENCRYPTION_CERT;
+import static uk.gov.ida.saml.core.test.TestCertificateStrings.TEST_RP_PUBLIC_SIGNING_CERT;
+
+@Path("/")
+public class TestMetatronResource {
+
+    @GET
+    @Path("/{entity-id}")
```

Comment:
Should the path be `"metadata/{entity-id}"? That's what got merged recently or are these tests set up different?

Reply:
The tests are set up to use the uri of the test resource so that's why it works anyway but I've changed it so that it documents things (like Metatron's endpoint) better.